hls4ml/writer/vivado_writer.py

```diff
@@ -293,24 +348,26 @@ def write_test_bench(self, model):
                 newline = line.replace('myproject', model.config.get_project_name())
             elif '//hls-fpga-machine-learning insert data' in line:
                 newline = line
+                newline += '      std::vector<float>::const_iterator in_begin = in.begin();\n'
```

Comment:
if you want `const_iterator` shouldn't you use `cbegin()`?

(I know I'm nitpicking here, but I have to comment on something or it will look like I didn't even go through the code :) )

Reply:
Haha sure, [begin() also returns a `const_iterator`](https://en.cppreference.com/w/cpp/container/vector/begin) but indeed `cbegin` would be more explicit.